README.md

```diff
@@ -10,3 +10,14 @@ might help with the notation for mathematical expressions.
 $G_1=(V_1 , E_1)$ is isomorphic to $G_2 = (V_2, E_2)$ if there exists a
 one-to-one and onto function (bijection) $f: V_1 \rightarrow V_2$ such that $(u,v)
 \in E_1$ iff $(f(u),f(v)) \in E_2$.
+
+## Answer
+
+Let's assume we have two graphs A and B that are isomorphic based on the definition above.
+
+If the graphs are not completely connected, there must be some edge $w = (u,v) \notin E_1$.  This is possible since it does not contradict the definition of isomorphism.
```

Comment:
Not quite following you here. How does this not contradict the definition? Even if it doesn't, does it affect your assumption?

Reply:
Could I instead prove this by example, where I have two graphs that are not completely connected but are isomorphic?

Reply:
That sounds like it's going in the right direction!